docs/book/src/topics/workload-identity.md

```diff
@@ -1,21 +1,21 @@
 # Workload Identity
 
-Azure AD Workload identity is the next iteration of Azure AD Pod identity 
-that enables Kubernetes applications (e.g. CAPZ) to access Azure cloud 
+Azure AD Workload identity is the next iteration of Azure AD Pod identity
```

Comment:
See also #4776, which fixes more formatting issues and some grammar in this document.

Reply:
It might be worth to just include changes from #4776 into this PR to avoid merge conflicts.